api/src/routes/settings.test.ts

```diff
@@ -271,6 +271,59 @@ describe('settingRoutes', () => {
       });
     });
 
+    describe('/update-my-about', () => {
+      test('PUT updates the values in about settings', async () => {
+        const response = await superRequest('/update-my-about', {
+          method: 'PUT',
+          setCookies
+        }).send({
+          about: 'Teacher at freeCodeCamp',
+          name: 'Quincy Larson',
+          location: 'USA',
+          picture:
+            'https://cdn.freecodecamp.org/platform/english/images/quincy-larson-signature.svg'
+        });
+
+        expect(response.statusCode).toEqual(200);
+
+        expect(response.body).toEqual({
+          message: 'flash.updated-about-me',
+          type: 'success'
+        });
+
+        const user = await fastifyTestInstance?.prisma.user.findFirst({
+          where: { email: 'foo@bar.com' }
+        });
+
+        expect(user?.about).toEqual('Teacher at freeCodeCamp');
+        expect(user?.name).toEqual('Quincy Larson');
+        expect(user?.location).toEqual('USA');
+        expect(user?.picture).toEqual(
+          'https://cdn.freecodecamp.org/platform/english/images/quincy-larson-signature.svg'
+        );
+      });
+
+      test('PUT updates the values in about settings without image', async () => {
+        const response = await superRequest('/update-my-about', {
+          method: 'PUT',
+          setCookies
+        }).send({
+          about: 'Teacher at freeCodeCamp',
+          name: 'Quincy Larson',
+          location: 'USA',
+          // `new URL` throws if the image is an empty string, this checks if it doesn't throw if it is empty.
+          picture: ''
```

Comment:
```suggestion
          picture: ''invalid"
```
Try this. I think the handler will still throw.

Reply:
Had to remove it, because I don't want to throw error when campers try to update the picture, when it is an empty string.